cirq/api/google/v2/device.proto

```diff
@@ -93,16 +94,31 @@ message ArgumentRange {
 
 // A list of targets that are valid for a set of gates.
 // For instance, all qubit pairs that can be acted on by a 2-qubit gate
-message TargetList {
+message TargetSet {
+  enum TargetOrdering {
+    UNSPECIFIED = 0;
+
+    // Symmetric gates, any id order within each target is valid.
+    SYMMETRIC = 1;
+
+    // Asymmetric gates, the order of ids in each target is important.
+    // Only the order specified in each target is valid.
+    ASYMMETRIC = 2;
+
+    // All targets should be single qubits.  Any subset of these targets
```

Comment:
Might want to clarify here the difference between "targets" in this `TargetSet`, vs. "targets" of the gate, something like: "All targets in this TargetSet should be single qubits. The gate itself can be applied to any subset of these targets, in any order. This may be the case for measurement gates, for example, if we can measure any subset of qubits at once."